### DISCUSSION TITLE
No user and password fields on Elasticsearch datasource setup

### DISCUSSION THREAD

**Discussion by z0st** [Budibase/budibase#3701] [Help] [answered]
### Missing username and password fields
Hi everyone!

I see there are no fields to specify user and password on Elasticsearch datasource setup, I'm connecting to Elasticsearch though the REST datasource.
Any other solution for this problem?

Thanks in advance, Alex

**Comment by mjashanks** [comment, marked as answer]
Hi Alex.

Have you tried using the format `https://[username]:[password]@[host]:[port]`, in the URL field?
